BuildGuide.md

```diff
@@ -0,0 +1,126 @@
+# Build Guide
+
+***Disclaimer***
+
+While this guide is meant to give Iron180 owners a comprehensive overview of how to build a 180, please keep in mind that there are many variations/alternatives to some of the processes described here. At the end of the day, some of these choices come to preference, so do not worry if you see other individuals building boards in a different way. That being said, if you are more of a visual person and would rather see videos, we have linked a few links to build streams below.
+
+
+## Table of contents
+
+* [Parts check and PCB](#parts-check-and-pcb)
+* [Dissasembly](#dissasembly)
+* [Bumpons, Daughterboard, and Gaskets](#bumpons-daughterboard-and-gaskets)
+* [Stabilizers and Layout](#stabilizers-and-layout)
+* [Switches](#switches)
+* [Soldering](#soldering)
+* [Reassembly](#reassembly)
+
+## Parts check and PCB
+
+Before building the iron 180, it is important to take a few precautionary steps. Before you take anything apart or start building, we recommend that you double-check the [parts list](Iron180PartsList.md) to ensure that nothing is missing.
+
+You can also refer to the following photo as a parts reference throughout the build.
+
+![Parts](Photos/Iron180Photos/parts_mod.jpg)
```

Comment:
consider matching color code to part list.

Reply:
changed to 
```You can also refer to the following photo as the color-coded part list throughout the build```